src/interpreter/ByteCode.h

```diff
@@ -278,13 +278,21 @@ class LoadLiteral : public ByteCode {
 #endif
 };
 
-class NewTargetOperation : public ByteCode {
+class MetaPropertyOperation : public ByteCode {
 public:
-    NewTargetOperation(const ByteCodeLOC& loc, const size_t registerIndex)
-        : ByteCode(Opcode::NewTargetOperationOpcode, loc)
+    enum Type {
+        NewTarget,
+        ImportMeta
+    };
+
+    MetaPropertyOperation(const ByteCodeLOC& loc, Type type, const size_t registerIndex)
+        : ByteCode(Opcode::MetaPropertyOperationOpcode, loc)
+        , m_type(type)
         , m_registerIndex(registerIndex)
     {
     }
+
+    Type m_type;
     ByteCodeRegisterIndex m_registerIndex;
```

Comment:
`ByteCodeRegisterIndex` has 16 bit sized type, so would you please define `Type` value as a 16 bit sized enum for 32bit alignment?
Or boolean flag would be a good alternative.

Reply:
That's good idea it should use 1 bit